### PR TITLE
More SOLID questions

### PR DESCRIPTION
**Pull request by vvscode**

### CODE REVIEW

File: general-development/d-solid-en.json5 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    answers: [
+      { text: 'Development' },
+      { text: 'Domain' },
+      { text: 'Data driven' },
+      { text: 'Dependency inversion', correct: true },
+      { text: 'Delegates segragation' },
+    ],
+    comment: 'the <b>Dependency Inversion (one should "depend upon abstractions, [not] concretions.)</b> principle refers to a specific form of decoupling software modules.\
+     When following this principle, the conventional dependency relationships established from high-level, \
+     policy-setting modules to low-level, dependency modules are reversed, \
```
Comment (zenby) [inline general-development/d-solid-en.json5:16]: Why do you add `\` to the end of line sometimes?

File: general-development/d-solid-en.json5 [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  type: 'choose_options',
+  lang: 'en',
```
Comment (zenby) [inline general-development/d-solid-en.json5:3]: This line means that we can only use english language at this card?

Comment (vvscode) [reply general-development/d-solid-en.json5:3]: you could use any language. This line is a meta-information, wich says that we're using english, to allow filter cards by language

File: general-development/d-solid-en.json5 (hunk 1 above)
Comment (vvscode) [reply general-development/d-solid-en.json5:16]: read https://json5.org

**Review by zenby — APPROVED** [review]

File: general-development/d-solid-en.json5 (hunk 2 above)
Comment (zenby) [reply general-development/d-solid-en.json5:3]: I mean if we use for example two languages for learning languages won't here a problem that there is only one string can be but not the array of these languages like `tags` field here?
Or it a little bit overabundant now?

Comment (vvscode) [reply general-development/d-solid-en.json5:3]: Discussion is welcome

Comment (vvscode) [reply general-development/d-solid-en.json5:3]: https://github.com/memory-cards/rfcs/issues

**Pull request merged**